<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensitivitySettings</class>
 <widget class="QWidget" name="SensitivitySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MouseSensitivitySettings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="resetToDefaultButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="sensitivityLabel">
       <property name="text">
        <string>Mouse Sensitivity: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="testSensitivityButton">
       <property name="text">
        <string>Test for 5 seconds</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="applySensitivity">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sensitivitySpinBox"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QSlider" name="sensitivitySlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
